### PR TITLE
Update capture.md

### PR DESCRIPTION
**Pull request by libbyhopfauf**
This update includes reorganizing the capture instructions to include a section for CLI vs. GUI. The CLI instructions were also broken down to include a section explaining how to build a CLI string, examples of frequently used capture commands and a list of the full commands (the last of which still needs to be updated and examples need to be added).

### CODE REVIEW

File: docs/_sections/capture.md [hunk 1]
```diff
@@ -3,4 +3,107 @@ layout: post
...
+
+4.) If you only have one device connected, dvrescue will automatically select it (device 0). If you have more than one device connected, add the device number ```device://1``` (for the corresponding deck you want to use for capture). Example (using the deck listed in step 1): ```dvrescue device://0``` 
+
+5.) If you want dvrescue to rewind and recapture when it encounters an issue with a frame type ```--rewind-count``` and the number of attempts you want it to make before it moves on. Example if you wanted dvrescue to rewind and recapture 3 times: ```--rewind-count 3```
+
+6.) Type ```-m``` to note you want to create a file.
+
+7.) Type the files name. Add a space. Example: ```filename.dv```
+
+8.) Type ```-m``` to note you want to merge the multiple capture attempts for problematic frames (dvrescue will automatically rewind and try to capture the frame again when it encouters a problematic frame).
```
Comment (dericed) [inline docs/_sections/capture.md:34]: this `-m` also needs an output `-` in this case, so the total would be `-m -`

File: docs/_sections/capture.md [hunk 2]
```diff
@@ -3,4 +3,107 @@ layout: post
...
+  
+**Basic useage:** <br />
+Below is the order you should type the commands for capturing using the CLI.
+
+1.) In the Terminal run ```dvrescue --list_devices``` to display the names of the devices connected to your computer. Note the name and device number of the one you want to use for capture. For example: <br />
+<img width="603" alt="Screenshot 2024-07-29 at 11 56 07 AM" src="https://github.com/user-attachments/assets/1ce94115-f6fc-467f-9b22-cb4e80763909">
+
+2.) You can either type ```dvrescue``` at the start of the command or include the path to the version of dvrescue you want to use. Example: ```/usr/local/bin/dvrescue```
+
+4.) If you only have one device connected, dvrescue will automatically select it (device 0). If you have more than one device connected, add the device number ```device://1``` (for the corresponding deck you want to use for capture). Example (using the deck listed in step 1): ```dvrescue device://0```
```
Comment (dericed) [inline docs/_sections/capture.md:26]: hmmm, don't you have the full guid in your result such as
```
$ dvrescue --list_devices
90:a1838cb4:00000000: UltraStudio 4K Mini [DeckLink]
```

File: docs/_sections/capture.md [hunk 3]
```diff
@@ -3,4 +3,107 @@ layout: post
...
+Is the default if no --cmd option is provided.<br />
+Usable only if input is a device.<br />
+Needs to be followed by the path and name for the output file.
+
+**--in-control** = Include an integrated command line input for controling the input.<br />
+Usable only if input is a device.
+
+**--list_devices** = List detected devices and their indices.
+
+**--statusl** = Provide the status (playing, stop...) of the input.<br />
```
Comment (dericed) [inline docs/_sections/capture.md:72]: I think you're testing against a pretty old CLI, since the statusl typo was fixed long ago.

Comment (libbyhopfauf) [reply docs/_sections/capture.md:72]: LOL my bad, this was actually copied out of the old documentation. My CLI is up to date. I just didn't have a deck to test haha.

Comment (libbyhopfauf) [reply docs/_sections/capture.md]: I changed it to have the latest version of dvrescue -h

File: docs/_sections/capture.md (hunk 2 above)
Comment (libbyhopfauf) [reply docs/_sections/capture.md:26]: Whoops, looks like that was an old example. Will update. So people need to add full number if they have more than one device plugged in right?

Comment (libbyhopfauf) [reply docs/_sections/capture.md]: @dericed See latest additions to the pull request for changes :)

**Comment by libbyhopfauf** [comment]
@dericed I made all of your suggested changes and made sure all of the latest updates are in there. But them me know if you catch anything else I should change before merging this in :)

**Comment by libbyhopfauf** [comment]
@dericed can this be merged?

**Comment by libbyhopfauf** [comment]
So, I'm hitting a little bit of a brick wall trying to make the CLI documentation accurate. These commands work: 

**Capture from a specific device:**
```dvrescue device://0xdevicename -m /pathtofile/filename.dv``` 
For example:
```dvrescue device://0x800460104ce2bad -m /Users/libbyshopfauf/Desktop/12345.dv```

**Capture from a specific device and show a preview window:** 
```dvrescue device://0xdevicename -m /pathtofile/filename.dv -m - | ffplay -``` 
For example:
```dvrescue device://0x800460104ce2bad -m /Users/libbyshopfauf/Desktop/12345.dv -m - | ffplay -```

These commands do not:
**Capture from a specific device using rewind, recapture and merge (you can only specify a number of times to rewind and recapture if you specify a device):**
```dvrescue device://0xdevicename --rewind-count # -m filename.dv -m -``` 
For example:
```dvrescue device://0x800460104ce2bad --rewind-count 3 -m /Users/libbyshopfauf/Desktop/12345.dv -m -```

**Capture from a specific device using rewind, recapture and merge (3 attempts), show a preview window (with only one deck connected):**
```dvrescue --rewind-count 3 -m filename.dv -m - | ffplay -```

**Basic capture (with only one deck connected):**
```dvrescue --rewind-count 3 -m filename.dv -m - | ffplay -```
</details>

**Comment by libbyhopfauf** [comment]
@dericed I just saw #914 regarding rewind and merge. So that might explain why it seemed like rewind and recapture was working even when I didn't have it as part of the command.

**Comment by libbyhopfauf** [comment]
@dericed this is ready to be merged! I updated the images to reflect how the GUI looks and what information is present for capture in the latest build. I also tested and updated all of the CLI commands and examples. Everything works and is good to go!

**Pull request merged**